rkt/pods.go

```diff
@@ -897,8 +896,8 @@ func (p *pod) getApps() (schema.AppList, error) {
 	if err != nil {
 		return nil, err
 	}
-	var pm *schema.PodManifest
-	if err = json.Unmarshal(pmb, &pm); err != nil {
+	pm := new(schema.PodManifest)
+	if err = pm.UnmarshalJSON(pmb); err != nil {
```

Comment:
I changed this because earlier in the file, in `getApps` it was doing it this way so I want to make it perform the same task the same way. But your comment got me looking at it a bit closer and I've now removed the body of `getAppCount` completely and reuse `getApps.` Thanks!